src/snippets/article-vor-basic.v1.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Article VoR snippet (basic)
+type: object
+allOf:
+  - $ref: article.v1.yaml
+  - properties:
+        status:
+            type: string
+            enum:
+              - vor
+        impactStatement:
+            type: string
+            minLength: 1
+        figuresPdf:
+            type: string
+            format: uri
+        image:
+            type: object
+            properties:
+                thumbnail:
+                    $ref: ../misc/image.v1.yaml
```

Comment:
this contains everything from `src/snippets/article-vor.v1.yaml` but without the `required` property. If this is set in `src/snippets/article-vor.v1.yaml` I am finding it difficult to override in the full article view where `thumbnail` may not be present but either `thumbnail` and/or `social` should be expected.